CONTRIBUTING.md

```diff
@@ -0,0 +1,121 @@
+
+
+#### How Do I Submit a Good Bug Report?
+
+We use GitHub issues to track bugs and errors. If you run into an issue with the project:
+
+- Open an [Issue](https://github.com/CleanCut/green/issues/new).
+- Explain the behavior you would expect and the actual behavior.
+- Please provide as much context as possible and describe the *reproduction steps* that someone else can follow to recreate the issue on their own. This usually includes your code. For good bug reports you should isolate the problem and create a reduced test case.
+- Provide the information you collected in the previous section.
+
+Once it's filed:
+
+- The project team will label the issue accordingly.
+- A team member will try to reproduce the issue with your provided steps. If there are no reproduction steps or no obvious way to reproduce the issue, the team will ask you for those steps and mark the issue as `needs-repro`. Bugs with the `needs-repro` tag will not be addressed until they are reproduced.
+
+
+
+### Suggesting Enhancements
+
+Follow these guidelines for suggesting enhancements.
+
+
+#### Before Submitting an Enhancement
+
+- Make sure that you are using the latest version of green.
+- Read the [documentation](#i-have-a-question) carefully and find out if the functionality is already covered, maybe by an individual configuration.
+- Perform a search of [issues](https://github.com/CleanCut/green/issues) and [discussions](https://github.com/CleanCut/green/discussions) to see if the enhancement has already been suggested. If it has, add a comment to the existing issue instead of opening a new one.
+- Find out whether your idea fits with the scope and aims of the project. It's up to you to make a strong case to convince the project's developers of the merits of this feature. Keep in mind that we want features that will be useful to the majority of our users and not just a small subset. If you're just targeting a minority of users, consider maintaining your own fork.
+
+
+#### How Do I Submit a Good Enhancement Suggestion?
+
+Enhancement suggestions are tracked as [GitHub Discussions](/discussions).
+
+- Use a **clear and descriptive title** for the issue to identify the suggestion.
+- Provide a **step-by-step description of the suggested enhancement** in as many details as possible.
+- **Describe the current behavior** and **explain which behavior you expected to see instead** and why. At this point you can also tell which alternatives do not work for you.
+- You may want to **include screenshots or screen capture videos** which help you demonstrate the steps or point out the part which the suggestion is related to. 
+- **Explain why this enhancement would be useful** to most users. You may also want to point out the other projects that solved it better and which could serve as inspiration.
+
+
+
+### Your First Code Contribution
+
+
+### Improving The Documentation
+
+
+## Styleguides
+### Commit Messages
+
```

Comment:
For each of these sections, let's either add some text to them, or remove them. Either way.